utils/RequestUtil.js

```diff
@@ -73,20 +73,23 @@ var RequestUtil = {
   json: function (options) {
     // Default assign options to empty object
     options || (options = {});
+    var requestID = options.url;
```

Comment:
@rcorral @MatApple I just pushed a commit which constructs the ID based on the properties `contentType`, `data`, `headers`, `method`, `url`. Let me know if you think this is sufficient!